.circleci/config.yml

```diff
@@ -8,7 +8,7 @@ executors:
     resource_class: large
   linux-amd64:
     machine:
-      image: ubuntu-2004:202107-02
+      image: default
```

Comment:
Because CircleCI deprecated such images: https://discuss.circleci.com/t/linux-image-deprecations-and-eol-for-2024/50177
I've already had to do this for Chronograf: https://app.circleci.com/pipelines/github/influxdata/chronograf/1682/workflows/f3c358d2-f591-452e-a41c-e00eaf9674a5/jobs/20110

Reply:
TBH I would prefer this being an own PR but as I'm not familiar with the kapacitor rules and habits I'm not insisting on it.

Reply:
Sure, it would be best to fix it separately. I need this in the PR to CircleCI build the PR.
Me or @bednar will create a separate PR.

Reply:
This executor was removed in PR #2835 because we are using the `cross-builder` across all stages of our pipeline.